src/windows7.rs

```diff
@@ -27,7 +27,9 @@ pub fn getrandom_inner(dest: &mut [MaybeUninit<u8>]) -> Result<(), Error> {
     // Prevent overflow of u32
     let chunk_size = usize::try_from(i32::MAX).expect("Windows does not support 16-bit targets");
     for chunk in dest.chunks_mut(chunk_size) {
-        let ret = unsafe { RtlGenRandom(chunk.as_mut_ptr().cast::<c_void>(), chunk.len() as u32) };
+        #[allow(clippy::cast_possible_truncation)]
+        let chunk_len = chunk.len() as u32;
```

Comment:
You can write it as `chunk.len().try_into().expect("chunk size is bounded by i32::MAX")`. Compiler is [able](https://rust.godbolt.org/z/e4Yqf53bj) to properly remove such panic.